frontend/src/app/services/plants.service.ts

```diff
@@ -391,6 +392,9 @@ export class PlantsService extends BaseService {
    * @param plants new plants.
    */
   private updatePlantsList(plants: Plant[]): void {
+    // handle case where plants were removed from server copy
+    this.plants = this.plants.filter(plant => plants.some(p => p.id === plant.id));
```

Comment:
fix for deleted plant not really getting deleted on the frontend without a refresh

Reply:
yea, i remember this